openshift/README.md

```diff
@@ -122,5 +122,5 @@ oc -n <namespace> rollout latest dc/<buildname>-master
 As of this time, we do not automatically clean up resources generated by a Pull Request once it has been accepted and merged in. This is still a manual process. Our PR deployments are all named in the format "pr-###", where the ### is the number of the specific PR. In order to clear all resources for a specific PR, run the following two commands to delete all relevant resources from the Openshift project (replacing `PRNUMBER` with the appropriate number):
 
 ```sh
-oc delete all -n <namespace> --selector app=clogs-pr-<PRNUMBER>
+oc delete all -n <namespace> --selector app=common-logging-<job name>
```

Comment:
Is this correct? PR-8 still has its pods with the label `app: clogs-pr-8`.

Reply:
ha, that was me just using the incorrect APP_NAME on the command line...